Source/WebCore/page/Navigator.cpp

```diff
@@ -233,6 +234,25 @@ void Navigator::showShareData(ExceptionOr<ShareDataWithParsedURL&> readData, Ref
     });
 }
 
+#if PLATFORM(COCOA) || ENABLE(PDFJS)
+// https://html.spec.whatwg.org/multipage/system-state.html#pdf-viewing-support
+// Section 8.9.1.6 states that if pdfViewerEnabled is true, we must return a list
+// of exactly five PDF view plugins, in a particular order.
+constexpr size_t dummyPDFPluginCount = 5;
+
+static const Vector<String>& dummyPDFPluginNames()
+{
+    static NeverDestroyed<Vector<String>> dummyPluginNames(std::initializer_list<String> {
+        String { "PDF Viewer"_s },
+        String { "Chrome PDF Viewer"_s },
+        String { "Chromium PDF Viewer"_s },
+        String { "Microsoft Edge PDF Viewer"_s },
+        builtInPDFPluginName(),
```

Comment:
I don't think we should be using `builtInPDFPluginName()` since this is a translated string. The spec says to use "WebKit built-in PDF". This would cause us to no longer be compliant on non-US systems.

Reply:
Okay, I'll change that. Then I can remove the LocalizedString changes I made, too.